config/management/network-address-encryption/src/lib.rs

```diff
@@ -89,30 +89,39 @@ impl Encryptor {
             .keys
             .get(&keys.current)
             .ok_or_else(|| Error::VersionNotFound(keys.current))?;
-        let raw_addr = std::convert::TryFrom::<&NetworkAddress>::try_from(network_address)?;
-        // TODO(davidiw) soon we'll take in a set, for now we assume there's only one addr
-        let encrypted_key =
-            EncNetworkAddress::encrypt(raw_addr, key, keys.current, &account, seq_num, 0);
-        lcs::to_bytes(&encrypted_key).map_err(|e| e.into())
+        let mut enc_addrs = Vec::new();
+        for (idx, addr) in network_addresses.iter().enumerate() {
+            let raw_addr = std::convert::TryFrom::<&NetworkAddress>::try_from(addr)?;
+            let idxu32 = idx as u32;
+            let enc_addr =
+                EncNetworkAddress::encrypt(raw_addr, key, keys.current, &account, seq_num, idxu32);
```

Comment:
why not idx as u32

Reply:
it becomes > 100 characters resulting in a lot of new lines